rllib/models/modelv2.py

```diff
@@ -246,35 +246,24 @@ def from_batch(self, train_batch: SampleBatch,
             i += 1
         return self.__call__(input_dict, states, train_batch.get("seq_lens"))
 
-    def get_view_requirements(
-            self, is_training: bool = False) -> Dict[str, ViewRequirement]:
-        """Returns a list of ViewRequirements for this Model (or None).
+    def get_view_requirements(self) -> Dict[str, ViewRequirement]:
+        """Returns a dict of ViewRequirements for this Model.
 
         Note: This is an experimental API method.
 
-        A ViewRequirement object tells the caller of this Model, which
-        data at which timesteps are needed by this Model. This could be a
-        sequence of past observations, internal-states, previous rewards, or
-        other episode data/previous model outputs.
-
-        Args:
-            is_training (bool): Whether the returned requirements are for
-                training or inference (default).
+        The view requirements dict is used to generate input_dicts and
+        train batches for 1) action computations, 2) postprocessing, and 3)
+        generating training batches.
 
         Returns:
-            Dict[str, ViewRequirement]: The view requirements as a dict mapping
-                column names e.g. "obs" to config dicts containing supported
-                fields.
-                TODO: (sven) Currently only `timesteps==0` can be setup.
+            Dict[str, ViewRequirement]: The view requirements dict, mapping
+                each view key (which will be available in input_dicts) to
+                an underlying requirement (actual data, timestep shift, etc..).
         """
         # Default implementation for simple RL model:
         # Single requirement: Pass current obs as input.
         return {
-            SampleBatch.CUR_OBS: ViewRequirement(timesteps=0),
-            SampleBatch.PREV_ACTIONS: ViewRequirement(
-                SampleBatch.ACTIONS, timesteps=-1),
-            SampleBatch.PREV_REWARDS: ViewRequirement(
-                SampleBatch.REWARDS, timesteps=-1),
+            SampleBatch.OBS: ViewRequirement(),
```

Comment:
Does it make sense to pass timesteps as well? `timesteps=1`?

Reply:
actually, timesteps is obsolete. Will fix this.